app/models/scaptimony/arf_report.rb

```diff
@@ -69,6 +68,10 @@ def delete
       end
     end
 
+    def assetable
```

Comment:
Won't work with belongs_to as it doesn't support :through - refactored to delegate.